src/Razor/src/Microsoft.AspNetCore.Razor.Language/Extensions/DesignTimeDirectiveTargetExtension.cs

```diff
@@ -81,7 +81,14 @@ private void WriteDesignTimeDirectiveToken(CodeRenderingContext context, Directi
                                 .WriteStartAssignment(TypeHelper)
                                 .Write("default(")
                                 .Write(node.Content)
-                                .WriteLine(");");
+                                .Write(")");
```

Comment:
While we're here, we could simplify this too. This could be `default` instead of `default(Foo)`.